flink-connector-kudu/src/main/java/org/apache/flink/connectors/kudu/streaming/KuduSink.java

```diff
@@ -97,6 +97,21 @@ public void invoke(IN value) throws Exception {
         }
     }
 
+    /**
+     * new invoke api
+     * @param value input
+     * @param context runtime context
+     * @throws Exception
+     */
+    @Override
+    public void invoke(IN value, Context context) throws Exception {
```

Comment:
Code duplication, this should invoke the other method or vice versa

Reply:
done